src/Azure.Functions.Cli/Interfaces/IDurableManager.cs

```diff
@@ -0,0 +1,21 @@
+﻿using System.Threading.Tasks;
+
+namespace Azure.Functions.Cli.Interfaces
+{
+    internal interface IDurableManager
+    {
+        Task DeleteHistory();
+
+        Task GetHistory(string instanceId);
+
+        Task GetRuntimeStatus(string instanceId, bool getAllExecutions);
+
+        Task RaiseEvent(string instanceId, string eventName, object eventData);
+
+        Task Rewind(string instanceId, string reason);
+
+        Task StartNew(string functionName, string version, string instanceId, object input);
+
+        Task Terminate(string instanceId, string reason);                
+    }
```

Comment:
Two commands I think we should add, both of which are new in **DurableTask.AzureStorage v1.3.2**):
* **GetInstances**: Internally gets the status of all instances with paging (the longest overload of `AzureStorageOrchestrationService.GetOrchestrationStateAsync`). By default, we should get all instances (from `DateTime.MinValue` to `DateTime.MaxValue`) and specify `top` as `10`.
* **PurgeHistory**: Internally calls `AzureStorageOrchestrationService.PurgeOrchestrationHistoryAsync`.